src/Text/Layout/Table.hs

```diff
@@ -248,21 +251,21 @@ tableLines specs TableStyle { .. } header rowGroups =
     headerSepLine = hLineDetail headerSepH headerSepLC headerSepC headerSepRC $ fakeColumns headerSepH
 
     -- Vertical content lines
-    rowGroupLines =
-        intercalate [groupSepLine] $ map (map (hLineContent groupV) . applyRowMods . rows) rowGroups
+    rowGroupLines = maybe concat (\sep -> intercalate [sep]) groupSepLine
```

Comment:
Please name the lambda argument `seps`, `lineSep`, `l`, or `line` in order to avoid confusion with horizontal separators.